src/Symfony/Component/DomCrawler/AbstractUriElement.php

```diff
@@ -70,7 +70,7 @@ public function getNode()
      */
     public function getMethod()
     {
-        return $this->method;
+        return $this->method ?? 'GET';
```

Comment:
Must return string because of https://github.com/symfony/symfony/blob/8496003634906d220ee1bc95f1547ced64545ffa/src/Symfony/Component/BrowserKit/Client.php#L292 where `request` has string type which would otherwise fail. So lets default back to `GET` which is in the constructor.

Reply:
Will it not be better to set the `null` replacement  directly on the `method` assignation in the `__construct` ? Line 46: `$this->method = $method ? strtoupper($method) : 'GET';`